src/js/serial_backend.js

```diff
@@ -466,14 +464,12 @@ function onConnect() {
         $('div#flashbutton a.flash_state').removeClass('active');
         $('div#flashbutton a.flash').removeClass('active');
     }
-    GUI.timeout_remove('connecting'); // kill connecting timer
```

Comment:
Why are you removing this? This is called _after_ the problem report dialog, so I don't think that this is the problem.